Add travis build step and fix javadoc formatting

csgregorian opened the pull request:
Fixed the javadoc formatting that gradle was complaining about and added a travis step that should build the whole thing every time.

xkrogen commented on .travis.yml:
```diff
@@ -6,4 +6,15 @@ jdk:
+jobs:
+  include:
+    - stage: build
+      script: "./gradlew build"
```
`./gradlew build` will be default also run the tests, I think we want `./gradlew build -x test`

xkrogen commented on dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditLogHiveTableParser.java:
```diff
@@ -25,7 +25,7 @@
  *   SELECT (timestamp - ${startTimestamp} AS relativeTimestamp, ugi, cmd, src, dst, ip
  *   FROM '${auditLogTableLocation}'
- *   WHERE timestamp >= ${startTimestamp} AND timestamp < ${endTimestamp}
+ *   WHERE timestamp &gt;= ${startTimestamp} AND timestamp &lt; ${endTimestamp}
```
This works, but looks ugly in the code. If you wrap the whole thing in an `{@code}` I think it will also work:
```
{@code
<pre>
  ...
</pre>
}
```
Here's a good discussion of the subject: https://reflectoring.io/howto-format-code-snippets-in-javadoc/

csgregorian commented:
Fixed changes and modified the build script a bit further: it runs a single build job (which doesn't depend on the `HADOOP_VERSION`) in parallel with two testing jobs (which each run on a different `HADOOP_VERSION`), as none of them depend on each other.
